dog.bat

```diff
@@ -2,8 +2,10 @@
 
 ECHO [Dog] WARNING: Ensure you have built using baufile.csx before building with this!
 
+scriptcs -I
+

```

Comment:
This is required if the scriptcs_packages folder has been removed. Otherwise the copy command will create the folder, preventing the implicit install (or any install) from taking place. While it does require executing baufile.csx I think leaving the separate install command in could prevent other problems or mistakes from causing failures in development.